test/00LessonZeroTest.js

```diff
@@ -7,8 +7,10 @@ function returnTrue() {
     return false;
 }
 
+// new comment line in this code
 
 // Tests
+//test update - attempt 2
 
 describe('#returnTrue()', function () {
     it('should return true', function () {
```

Comment:
Well I watched the lesson 0 video and got the pull request setup. I'm working on the pig latin homework now. I guess we will see how well its going soon enough.